### PR TITLE
Selection logic and styling

### PR DESCRIPTION
**Pull request by mttwc**

### CODE REVIEW

File: src/oneNotePicker.scss [hunk 1]
```diff
@@ -20,3 +20,8 @@
   float: right;
   margin-right: 5px;
 }
+
+.picker-selectedItem {
+  // Office UI Fabric specifies a hover which has more priority this, hence the !important
+  background-color: #D0D0D0 !important;
```
Comment (mttwc) [inline src/oneNotePicker.scss:26]: If anyone has any suggestions here ... :)

Comment (leakymemory) [inline src/oneNotePicker.scss]: Boo.  Since this is so scoped I can live with it.

Comment (leakymemory) [reply src/oneNotePicker.scss:26]: Boo. `!important`s drive me nuts. But, since this is so scoped I can live with it.

**Review by leakymemory — APPROVED** [review]

File: src/oneNotePicker.scss (hunk 1 above)
Comment (mttwc) [reply src/oneNotePicker.scss:26]: Yeah! We can't modify partner css files and hover is more scoped than non-hover, so I couldn't think of anything else :(

**Pull request merged**